src/wmtk/operations/tet_mesh/EdgeOperationData.cpp

```diff
@@ -49,17 +51,117 @@ std::array<Tuple, 2> EdgeOperationData::input_endpoints(const TetMesh& m) const
 
 std::vector<Tuple> EdgeOperationData::collapse_merged_ear_edges(const TetMesh& m) const
 {
```

Comment:
what happened with all of these impl? are they replaced somewhere?

Reply:
the ordering in these functions was important for attribute transfer BTW.

Reply:
these didn't compile before I commented them out